rules/go_rules.build_defs

```diff
@@ -4,19 +4,12 @@ Go has a strong built-in concept of packages so it's probably a good idea to mat
 rules to Go packages.
 """
 
-# This links all the .a files up one level. This is necessary for some Go tools to find them.
-_LINK_PKGS_CMD = ' '.join([
-    'find . -name "*.a" -print0 | sort -z | xargs -0 -n1 | while read FN; do',
-    'DN=${FN%/*}; BN=${FN##*/}; if [ "${DN##*/}" == "${BN%.a}" ]; then',
-    'ln -s "$TMP_DIR/$FN" "${DN%/*}"; fi; done'
-])
-
-_DEFAULT_GO_PATH = "$TMP_DIR:$TMP_DIR/src:$TMP_DIR/$PKG_DIR:$TMP_DIR/third_party/go:$TMP_DIR/third_party/"
 
 def go_library(name:str, srcs:list, asm_srcs:list=None, hdrs:list=None, out:str=None, deps:list=[],
                visibility:list=None, test_only:bool&testonly=False, complete:bool=True,
                _needs_transitive_deps=False, _all_srcs=False, cover:bool=True,
-               filter_srcs:bool=True, _link_private:bool=False, _link_extra:bool=True, _abi:str=False):
+               filter_srcs:bool=True, _link_private:bool=False, _link_extra:bool=True, _abi:str=False,
+               _generate_import_config:bool=True, _import_path:str= '', _pre_build=None):
```

Comment:
It might make sense to make `_import_path` a "public" argument: third-party packages will definitely have to set that manually.